pkg/virt-handler/device-manager/device_controller.go

```diff
@@ -191,7 +192,11 @@ func (c *DeviceController) updatePermittedHostDevicePlugins() []Device {
 	}
 
 	if c.virtConfig.PersistentReservationEnabled() {
-		permittedDevices = append(permittedDevices, NewSocketDevicePlugin(reservation.GetPrResourceName(), reservation.GetPrHelperSocketDir(), reservation.GetPrHelperSocket(), c.maxDevices))
+		d, err := NewSocketDevicePlugin(reservation.GetPrResourceName(), reservation.GetPrHelperSocketDir(), reservation.GetPrHelperSocket(), c.maxDevices, selinux.SELinuxExecutor{})
+		if err != nil {
+			log.Log.Reason(err).Errorf("failed to configure the desired mdev types, failed to get node details")
+		}
+		permittedDevices = append(permittedDevices, d)
```

Comment:
It's seems that d will be added even if NewSocketDevicePlugin fails, is that expected?

Reply:
Good catch!